src/plugins/comicsPlayer/plugin.js

```diff
@@ -259,7 +262,12 @@ class ArchiveSource {
         this.raw = await this.archive.getFilesArray();
         await this.archive.extractFiles();
 
-        const files = await this.archive.getFilesArray();
+        let files = await this.archive.getFilesArray();
+        files = files.filter((file) => {
+            const name = file.file.name;
+            const index = name.lastIndexOf('.');
+            return index !== 1 && supportedFormats.includes(name.slice(index + 1));
```

Comment:
Why is this checking `index !== 1`?

Reply:
To ignore files like `jpg`, `png` - `slice(-1 + 1)`

Reply:
Shouldn't it be `-1`?

Reply:
Oh, yes, it should be `-1` as in the original version https://github.com/jellyfin/jellyfin-web/issues/3352#issuecomment-1017757810